src/SumoCoders/FrameworkCoreBundle/Tests/Controller/DatePickerControllerTest.php

```diff
@@ -50,21 +41,9 @@ public function testIfSingleTextWithDatePickerAndDefaultDateRenderedCorrectly()
         $crawler = $this->getCrawlerForRequest('GET', '/_tests/datepicker');
 
         $element = $crawler->filter('input#form_date_example4');
```

Comment:
don't forget to adjust this id when shifting the ids in the code

Reply:
fixed 89a0a2e